zebra-rpc/src/methods.rs

```diff
@@ -74,6 +112,37 @@ impl Rpc for RpcImpl {
 
         Ok(response)
     }
+
+    fn get_block(&self, height: Height) -> BoxFuture<Result<GetBlock>> {
+        let mut state = self.state_service.clone();
+
+        async move {
+            let request = zebra_state::Request::Block(zebra_state::HashOrHeight::Height(height));
+            let response = state
+                .ready()
+                .await
+                .expect("State service should be always ready")
+                .call(request)
+                .await
+                .map_err(|error| Error {
+                    code: ErrorCode::ServerError(0),
+                    message: error.to_string(),
+                    data: None,
+                })?;
+
+            match response {
+                zebra_state::Response::Block(Some(block)) => Ok(GetBlock {
+                    data: hex::encode(block.to_string()),
```

Comment:
Bug: This uses the logging format for blocks:
https://github.com/ZcashFoundation/zebra/blob/30b5468f2fa37d39bf377a626bf11a4288734d5b/zebra-chain/src/block.rs#L56-63

We want the block data instead:
```suggestion
                    data: hex::encode(block.zcash_serialize_to_vec()),
```

Reply:
Thanks! I fixed this at https://github.com/ZcashFoundation/zebra/pull/3707/commits/1258d26c55992733a44a3cd07eda83023e5ef9a8